App/App.js

```diff
@@ -94,39 +96,58 @@ const App = {
    */
   displayList: async function({ tabsList }) {
     const tabListDomElement = document.querySelector('.tab-list');
-
-    if (!Array.isArray(tabsList)) {
-      tabsList = [tabsList];
-    }
-
     const currentWindowId = await this.getCurrentWindow();
 
-    tabsList.sort((a, b) => {
-      return (a.id === currentWindowId) ? -1 : 1;
-    })
+    // if (!Array.isArray(tabsList)) {
+    //   tabsList = [tabsList];
+    // }
```

Comment:
I bet you left it accidentally

--your friendly neighborhood

Reply:
Will remove, thanks!